pup.py

```diff
@@ -180,8 +180,9 @@ def make_producer(queue=None):
     try:
         timeout = aiohttp.ClientTimeout(total=60)
         async with aiohttp.ClientSession() as session:
-            async with session.post(configuration.INVENTORY_URL, data=json.dumps(post), headers=headers, timeout=timeout) as response:
-                if response.status != 200 and response.status != 201:
+            async with session.post(configuration.INVENTORY_URL, data=json.dumps([post]), headers=headers, timeout=timeout) as response:
+                response_json = await response.json()
+                if response_json['data'][0]['status'] != 200 and response_json['data'][0]['status'] != 201:
```

Comment:
can the inv service return a non-200 response without the expected json object? for example, if it gave a 503 or 404 back.

Reply:
If inventory status returns something else we'll just error that something went wrong and log it. We don't dig through the JSON if something went wrong like a 404 or 500. This latest commit handles that